openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -105,7 +107,9 @@ <h5 class="submission__answer__part__text__title">{% trans "The prompt for this
                                         data-preview="submission__{{ forloop.counter }}"
                                         aria-labelledby="submission__answer__part__text__title__{{ forloop.counter }}__{{ xblock_id }}"
                                         aria-describedby="submission__answer__tip__{{ xblock_id }}"
-                                        maxlength="100000"
+                                        {% if not forloop.first and not forloop.counter|divisibleby:2 %} rows="10" cols="50" {%endif%}
```

Comment:
Much better than the unpacking I proposed. This way we don't need to worry about new problem addition.
Thanks for this.